fa/homework/README.md

```diff
@@ -0,0 +1,157 @@
+# کار در خانه: چیزهای بیشتری به وب‌سایت اضافه کنیم!
+
+وبلاگ ما راه درازی را آمده است اما هنوز جا برای توسعه بیشتر، هست. در ادامه ما ویژگی‌هایی‌هایی را مانند ایجاد پیش‌نویس برای یک پست وبلاگی و نحوه انتشار آن‌ها، اضافه خواهیم کرد. علاوه بر این امکان پاک کردن پست‌هایی که لازم نداریم را هم ایجاد خواهیم کرد. بسیارعالی!
+
+## ذخیره کردن پست‌های جدید به صورت پیش‌نویس 
+
+در حال حاضر وقتی ما به کمک فرم *New post* یک پست وبلاگی جدید می‌سازیم، این پست مستقیماً منتشر می‌شود. برای آنکه این پست به صورت پیش‌نویس ذخیره شود عبارت زیر را در متدهای `post_new` و `post_edit` در فایل `blog/views.py` **حذف** کنید:
+
+```python
+post.published_date = timezone.now()
+```
+
+با این روش، پست‌های جدید به جای آنکه بلافاصله منتشر شوند، به عنوان پیش‌نویس ذخیره می‌شوند و بعداً می‌توانیم آن‌ها را بازبینی کنیم. چیزی که الان لازم داریم راهی است که پست‌های پیش‌نویس شده را لیست کنیم و انتشار دهیم، برویم سراغش! 
+
+## صفحه‌ای با لیستی از پست‌های منتشر نشده 
+
+بخش مربوط به کوئری‌ست‌ها را به یاد می‌آورید؟ ما ویویی به نام `post_list` ساخته‌ایم که فقط پست‌های منتشر شده را نمایش می‌دهد (پست‌هایی که مقدار `published_date` برای آن‌ها خالی نیست).
+
+حالا زمان آن است که کار مشابهی را برای پست‌های پیش‌نویس انجام دهیم.
+
+حالا می‌خواهیم در هدر فایل `blog/templates/blog/base.html` یک لینک اضافه کنیم. از آنجا که  نمی‌خواهیم لیست پست‌های پیش‌نویس را به همه نشان بدهیم، پس این لینک رادر عبارت کنترلی {% if user.is_authenticated %} و دقیقاً بعد از دکمه مربوط به اضافه کردن پست جدید، اضافه می‌کنیم.
```

Comment:
> Could someone help me why one of the check is failed?

The [build failure was](https://travis-ci.org/github/DjangoGirls/tutorial-extensions/builds/761286734#L394):
```
error: error while generating page "homework/README.md": 
Template render error: (/home/travis/build/DjangoGirls/tutorial-extensions/fa/homework/README.md)
  parseIf: expected elif, else, or endif, got end of file
The command "gitbook build" exited with 1.
```
The template tag syntax of HonKit overlaps with the template tag syntax of Django, so sometimes when we mention Django template tags, we have to put `{% raw %}…{% endraw %}` around them, to keep HonKit from trying to evaluate them. I guess that might be the case here, too, so try this:
```suggestion
حالا می‌خواهیم در هدر فایل `blog/templates/blog/base.html` یک لینک اضافه کنیم. از آنجا که  نمی‌خواهیم لیست پست‌های پیش‌نویس را به همه نشان بدهیم، پس این لینک رادر عبارت کنترلی {% raw %}`{% if user.is_authenticated %}`{% endraw %} و دقیقاً بعد از دکمه مربوط به اضافه کردن پست جدید، اضافه می‌کنیم.
```

Reply:
@das-g  After I've made these changes, should I create a new pull request?

Reply:
>  After I've made these changes, should I create a new pull request?

No, just tell us when this Pull Request here is ready for review.